validator/src/main/java/com/google/daq/mqtt/validator/ReportingDevice.java

```diff
@@ -200,35 +208,57 @@ public Entry getErrorStatus() {
     if (errors.isEmpty()) {
       return null;
     }
+    return errors.size() == 1 ? makeEntry(errors.get(0)) : makeCompoundEntry(errors);
+  }
+
+  private Entry makeCompoundEntry(List<Exception> erx) {
```

Comment:
Could we have docstrings that give example input/output here? github doesn't lend itself to jumping around very quickly to piece it together.